docs/appendices/release-notes/5.5.0.rst

```diff
@@ -73,6 +73,11 @@ SQL Statements
     my_column text index using fulltext with (analyzer = 'myanalyzer')
 
 
+- ``CREATE TABLE`` statements now allow to define column ``CHECK`` constraints
```

Comment:
```suggestion
- Allowed ``CREATE TABLE`` statements to define column ``CHECK`` constraints
```
Maybe it's better this way to use past tense, as we usually do?

Reply:
Rephrased it a bit